spec/javascript/support/puppeteer/steps/applications.js

```diff
@@ -0,0 +1,21 @@
+import puppeteer from 'puppeteer'
+
+import sharedSteps from './sharedSteps'
+import { HEADLESS } from '../consts'
+
+export const applicationRouteCheck = async (type, id) => {
+  let browser = await puppeteer.launch({ headless: HEADLESS })
```

Comment:
can you add a comment here explaining what this helper function is for. Application route check isn't 100% self-explanatory to me